app/helpers/results_helper.rb

```diff
@@ -24,8 +24,13 @@ def render_series_facet_link(doc)
 
   def render_repository_facet_link(doc)
     repos_id = Solrizer.solr_name("repository", :stored_sortable)
+    repository_label repositories.find{|key,hash| hash["admin_code"] == doc}[1]["url"]
+  end
+  
+  def render_repository_link(doc)
```

Comment:
@chrpr Please provide specs for these functions as they are the reason code coverage is going down: https://coveralls.io/builds/2122685/source?filename=app%2Fhelpers%2Fresults_helper.rb